src/acquisition/covid_hosp/common/test_utils.py

```diff
@@ -57,3 +82,17 @@ def load_sample_metadata(self, metadata_name='metadata.csv'):
 
   def load_sample_dataset(self, dataset_name='dataset.csv'):
     return pandas.read_csv(self.data_dir / dataset_name, dtype=str)
+
+  def truncate_tables(self, database, tables):
+    # use the local instance of the Epidata API
+    Epidata.BASE_URL = 'http://delphi_web_epidata/epidata/api.php'
+
+    # use the local instance of the epidata database
+    secrets.db.host = 'delphi_database_epidata'
+    secrets.db.epi = ('user', 'pass')
```

Comment:
this is a little heavy-handed for a table truncation method...  this stuff would be more appropriate in the `setUp()` method of a subclass of `unittest.TestCase` that can then be subclassed by each of `integrations/acquisition/covid_hosp/*/test_scenarios.py:AcquisitionTests`